lib/chat/bloc/provider/ChatProvider.dart

```diff
@@ -33,11 +33,15 @@ class _Constants {
   static const String typeValueMultiChoice = "com.wearemobilefirst.MultiChoice";
 }
 
-class _Strings {
-  static const String summaryTitleValue = "Chat summary value";
-  static const String summaryTitleLabel = "Chat summary";
-  static const String summaryActionButtonText = "Confirm";
-  static const String summaryError = "Provided summary is not correct";
+class _LocalisedStrings {
```

Comment:
this is not part of the farmsmart app. this is also not in any spec, its a screen Alex has made up for debugging. lets remove these as we don´t want them to translate debug.